lib/ASTGen/Sources/ASTGen/Diagnostics.swift

```diff
@@ -458,3 +456,111 @@ public func renterQueuedDiagnostics(
         allocateUTF8String(renderedStr)
   }
 }
+
+
+// MARK: - ASTGen diagnostics
```

Comment:
`ASTGenError.swift` or `ASTGenDiagnostics.swift`?

Reply:
IMO `ASTGenDiagnostics.swift` is self-explanatory and more prudent . I also propose that we rename  `Diagnostics.swift` to either `DiagnosticsBridging.swift` or `DiagnosticsBridge.swift` because it mostly, if not exclusively, deals with emitting SwiftSyntax diagnostics via the C++ engine.

Reply:
Let’s do `ASTGenDiagnostics.swift`. That way we don’t need to rename the file if we introduce ASTGen warnings at some point.



> I also propose that we rename `Diagnostics.swift` to either `DiagnosticsBridging.swift`.

Very good idea 👍🏽